benchmarks/openthread-2019-12-23/build.sh

```diff
@@ -14,7 +14,7 @@
 # limitations under the License.
 
 cd openthread
-git checkout 5b0af03afb8e70e8216f69623bd18bcd3d4b8b43
+git checkout 5bffd8bc4dadfedb6ce42c534e4ea33b2a3d1f10
```

Comment:
Just to fix CI. See https://github.com/google/fuzzbench/pull/728#issuecomment-691801723

Reply:
This breaks KLEE, so reverted, also unrelated to this CL. These afl failures with openthread have been opening for a while (probably after we upgraded benchmark to work with newer clang (due to clang coverage tools).